docs/core/tools/index.md

```diff
@@ -50,6 +50,15 @@ The following commands are installed by default:
 - [msbuild](dotnet-msbuild.md)
 - [dotnet install script](dotnet-install-script.md)
 
+**Commands for managing tools**
```

Comment:
Some of these were really just added in 2.1, so they don't apply to all 2.x

I've been removing tabs from the CLI reference but this article hasn't been updated in a while.
If we update ms.date, we should make sure that other commands are also not missing since 3.x is not here yet. 
I've also agreed with Kathleen that we could start removing 1.x info from the CLI reference.

Reply:
Is there a list of commands added in 3.0/3.1?

Reply:
Reviewed the what's new docs, looks like the only new commands were local tool restore and run commands.  Added those with "available since 3.0", removed 1.0, added "applies to 2.1 and later"

Reply:
The PR is now dependent on #17065 being merged first, since it links to an anchor added in that PR.